docs/reference/technology/firmware_update.md

```diff
@@ -54,19 +58,22 @@ For more information about [managed and unmanaged bootloaders, and how to create
 ### Porting a bootloader
 
 If you're interested in porting the Update client and bootloader to new hardware, please [review the porting section](https://www.pelion.com/docs/device-management/current/porting/porting-the-device-management-update-client.html).
-
+>Can this link just point to the DM-side porting docs?
```

Comment:
If this is all about porting a bootloader with Update client and we have a guide for that, I think we should link to that guide as opposed to generic porting.

Reply:
The Update client portion of the porting guide might go--if you port Client, Update client is included with it. So there's no point in having separate docs for 'porting' it. There might be additional material on *configuration*, if necessary. I'll change this link as necessary.